torch/csrc/api/include/torch/fft.h

```diff
@@ -5,25 +5,113 @@
 namespace torch {
 namespace fft {
 
-#ifndef DOXYGEN_SHOULD_SKIP_THIS
-namespace detail {
+/// Computes the 1 dimensional fast fourier transform over a given axis
+///
+/// \see https://pytorch.org/docs/master/fft.html#torch.fft.fft
```

Comment:
nit: "Computes the one-dimensional fast Fourier transform over a given axis. See <url>."